base/multi.jl

```diff
@@ -1338,7 +1338,7 @@ pmap(f) = f()
 # rsym(n) = (a=rand(n,n);a*a')
 # L = {rsym(200),rsym(1000),rsym(200),rsym(1000),rsym(200),rsym(1000),rsym(200),rsym(1000)};
 # pmap(eig, L);
-function pmap(f, lsts...; err_retry=true, err_stop=false)
+function pmap(f, lsts...; err_retry=true, err_stop=false, pids = Base.workers())
```

Comment:
Can be just `pids=workers()`, `multi.jl` is part of `Base` and since `workers` is an exported function, there is no need to qualify it with `Base` elsewhere too.